enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+information.
+
+## Motivation
+
+See summary above, motivation is to make the install experience clearer with less potential for misconfiguration and conflicting settings.
+
+### Goals
+
+- Post install the cluster can not only pull images from an alternate registry, but imagestreams can import from it, builds can pull from it, and the internal registry can perform pullthrough against it
+- Post install the registry CAs provided by the admin are populated into the configuration api that is intended for this purpose
+
+### Non-Goals
+
+- Refactoring the additionalTrustBundle field into many unique fields for different CA usages
+- Addressing the current proxy CA behavior in which the additionalTrustBundle is sometimes populated into a proxy configuration and other times not (depending on whether a proxy host is supplied)
+- Adding generic cluster configuration fields to the installer that are not required to bootstrap the cluster and ensure consistent use of cluster config apis
+
+
+## Proposal
+
+The installer should populate the 
+[image config resource](https://github.com/openshift/api/blob/master/config/v1/types_image.go) in the same way the 
+[product documentation](https://docs.openshift.com/container-platform/4.5/openshift_images/image-configuration.html#images-configuration-cas_image-configuration) 
+recommends.  The installer 
+can use the value of the additionalTrustBundle field to populate this api.  Similar to how the 
+value is used for the proxy config when a proxy host is supplied, the value can be used to 
+populate the image config resource when a mirror registry is in use.
+
+In addition, the downstream components that potentially communicate with the mirror registry should add logic which confirms they can successfully access the mirror registry.  Specifically, for any registry defined in the image content source policy, the components should be able to access the registry using the supplied CAs.  If they cannot, they should mark themselves degraded.
+
+Finally, if possible, the MCO mechanism that injects CAs into the nodes directly should be disabled when the image config api is populated, to ensure we do not have conflicting values being placed on the nodes.  The MCO mechanism should be a pure bootstrapping mechanism.  (That may already be the case today, I am not certain how that mechanism is implemented).
+
+### User Stories
+
+#### Story 1
+
+As an administrator installing a new cluster, I do not want to have to take additional actions on day 2 of my cluster installation or risk forgetting to do so only to have surprising failures later when the cluster is in use.  
+
+#### Story 2
+
+As an administrator of a cluster I do not want nodes on my cluster being configured with a CA that is different from what I configured the cluster to trust via the product documented apis, so that I am not confused by my cluster's behavior relative to what the configuration appears to be.
```

Comment:
Can you explain this a little bit more? I am not sure which api you are referring to here. and how the CAs ended up being different.

Reply:
i'm referring to the image CA config api.  The CAs end up different when you supply the additioanlTrustBundle which puts the CAs on the nodes, but you do not supply the image CA config, which means that you have two different sets of CAs in play:

1) the set the nodes are using when talking to registries (additionalTrustBundle CAs  + image config CAs)
2) the sets everything else that talks to registries is using (image config CAs)